src/metrics.ts

```diff
@@ -338,6 +338,13 @@ export function getMetrics(
       help: 'Total count of msg publish data.length bytes',
       labelNames: ['topic']
     }),
+    /** Total time in millisecond to publish a message */
+    msgPublishMs: register.histogram<{ topic: TopicLabel }>({
+      name: 'gossipsub_msg_publish_ms',
+      help: 'Total time in millisecond to publish a message',
```

Comment:
Prometheus standards strongly suggest to use standard metrics, in the case of time seconds: https://prometheus.io/docs/practices/naming/#base-units

Reply:
thanks @dapplion I created https://github.com/ChainSafe/js-libp2p-gossipsub/pull/457/files